packages/docs/src/pages/hooks.mdx

```diff
@@ -0,0 +1,22 @@
+import Layout from '../components/layout'
+
+export default Layout
+
+# Hooks
+
+If you are interested in learning more about how React Hooks work please see the official React docs [here](https://reactjs.org/docs/hooks-intro.html). 
+
+## `useThemeUI`
+
+To access the `theme` object directly in a component, you can use the `useThemeUI` hook.
+
+```jsx
+import React from 'react'
+import { useThemeUI } from 'theme-ui'
+
+export default props => {
+  const { theme } = useTheme
+  ...
```

Comment:
We should probably make this a comment so that it works when cut and pasted.

```suggestion
  // ...
```

Perhaps we could even display the theme by default and make the component:

```js
export default () => {
  const { theme } = useThemUI()
  
  return <pre>{JSON.stringify(theme, null, 2)}</pre>
}
```